web/router.js

```diff
@@ -191,4 +203,26 @@ router.afterEach((to) => {
   afterEachSetTitle(to);
 });
 
+function onErrorShowToast(err) {
+  // TODO: only show this for study request stuff!
```

Comment:
we should ensure that our backends are returning sensible errors here (but we can do that in a separate PR)